src/spikeinterface/extractors/neoextractors/blackrock.py

```diff
@@ -80,12 +82,27 @@ class BlackrockSortingExtractor(NeoBaseSortingExtractor):
 
     mode = "file"
     NeoRawIOClass = "BlackrockRawIO"
-    handle_spike_frame_directly = False
+    neo_returns_timestamps = True
     name = "blackrock"
 
-    def __init__(self, file_path, sampling_frequency=None):
+    def __init__(self, file_path, stream_index: Optional[int] = None):
```

Comment:
I am not sure to be 100% happy with this because if you have a blackrock- spike files (nev) without any otehr file.
Then you cannot load it. And this stream_index expose to users seams a bit complicated.

Reply:
Wheras if the sampling_frequency is exposed then you will always be able to load it.

Reply:
I need to check this again, this was a month ago and it is not fresh in my head. I will check on it on the weekend. What you say makes sense at this level though.  Thanks for taking a look,  Sam.

Reply:
We all agree that `sampling_frequency` should be keept and have precedence.

Reply:
OK, what are the errors. Hierarchy:
1) Only one stream use it.
2) Multiple streams same sampling frequency, use that one.
3) ...

Reply:
Implemented in the last commit.